dtmi/com/microchip/pic18F57q84_cnano-1.json

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
Thanks for the submission. File names need to be in lowercase, there is a capital F in the current submission pic18**F**57q84_cnano-1.json -> pic18f57q84_cnano-1.json